worlds/tloz/__init__.py

```diff
@@ -168,9 +167,8 @@ def apply_base_patch(self, rom):
         # Remove map/compass check so they're always on
         # Removing a bit from the boss roars flags, so we can have more dungeon items. This allows us to
         # go past 0x1F items for dungeon items.
-        base_patch_location = os.path.dirname(__file__) + "/z1_base_patch.bsdiff4"
-        with open(base_patch_location, "rb") as base_patch:
-            rom_data = bsdiff4.patch(rom.read(), base_patch.read())
+        base_patch = get_data(__name__, os.path.dirname(__file__) + "/z1_base_patch.bsdiff4")
```

Comment:
os.path.join() should be used over an explicit "/"

Reply:
So I've pushed a commit to address but github's being a butt atm and not updating the PR. hopefully it shows up SOONtm

Reply:
....and 14 hours later it still isn't showing up. unsure what to do about this

Reply:
ok FINALLY got it to show up after merging in the newest main now that there was a new commit